092.save-raws-to-Rdata.Rmd

```diff
@@ -13,18 +13,25 @@ knit: (function(inputFile, encoding) {
 ```{r setup, include=FALSE}
 library(tidyverse)
 library(lubridate)
-library(rnaturalearth)
 source('utils/r-utils.R')
 theme_set(theme_bw() + theme(legend.title = element_blank()))
 ```
 
 
+```{r}
+# library(rnaturalearth)
+# world <- rnaturalearth::ne_countries(scale='medium', returnclass = 'sf')
+# save(world, file = 'Rdata/world.Rdata')
```

Comment:
Maybe add a comment that rnaturalearth does not work in the docker  (why does it not work?) and that is why we're loading it from Rdata

Reply:
managed to make it work!